hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/table/action/bootstrap/SparkBootstrapCommitActionExecutor.java

```diff
@@ -390,8 +390,15 @@ private BootstrapWriteStatus handleMetadataBootstrap(String srcPartitionPath, St
 
     TypedProperties properties = new TypedProperties();
     properties.putAll(config.getProps());
-    KeyGeneratorInterface keyGenerator  = (KeyGeneratorInterface) ReflectionUtils.loadClass(config.getBootstrapKeyGeneratorClass(),
-        properties);
+
+    KeyGeneratorInterface keyGenerator;
+    try {
+      keyGenerator = HoodieSparkKeyGeneratorFactory.createKeyGenerator(properties);
+    } catch (IOException e) {
+      LOG.error("Init keyGenerator failed ", e);
```

Comment:
Since we have thrown an exception, we may not need to add a log.

Reply:
> Since we have thrown an exception, we may not need to add a log.

removed